tests/console/irqbalance.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Package: irqbalance
+# Summary: Test irqbalance runs as service as well as standalone
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use warnings;
+use base "consoletest";
+use strict;
+use testapi qw(is_serial_terminal :DEFAULT);
+use utils qw(systemctl zypper_call clear_console);
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Return when balancing is ineffective on system with a single cpu
+    my $nproc = script_output('cat /proc/cpuinfo | grep processor | wc -l');
+    die('Balancing is ineffective on systems with a single cpu') unless($nproc>1);
```

Comment:
```suggestion
    die("Balancing is ineffective on systems with a single cpu (nproc=$nproc)") unless($nproc>1);
```

I'd also add `$nproc` to the error message. Like this you can immediately see if something goes wrong when determining the number of CPUs (e.g. `$nproc=0`).